### ISSUE TITLE
k8s_core_v1_secret resource leaks sensitive data

### ISSUE THREAD

**Issue by UncleIS** [mingfang/terraform-provider-k8s#12]
Hi! 

The provider looks great, thanks for the excellent work! 

There is one thing that bothers me, though: the provider leaks sensitive data in `k8s_core_v1_secret` resource:

```
Terraform will perform the following actions:                                                                                                                 
                                                                                                                                                              
  # k8s_core_v1_secret.app_saml will be created                                                                                                               
  + resource "k8s_core_v1_secret" "app_saml" {                                                                                                                
      + data        = {                                                                                                                                       
          + "metadata.xml" = "<My base64-encoded string here (from a data source of another provider)>"
          + "service.jks"  = "<My base64-encoded string here (from a data source of another provider)>"
        }
      + id          = (known after apply)
      + string_data = (known after apply)
      + type        = (known after apply)

      + metadata {
          + annotations                   = (known after apply)
          + creation_timestamp            = (known after apply)
          + deletion_grace_period_seconds = (known after apply)
          + deletion_timestamp            = (known after apply)
          + labels                        = (known after apply)
          + name                          = "app-saml"
          + namespace                     = "app"
          + resource_version              = (known after apply)
          + self_link                     = (known after apply)
          + uid                           = (known after apply)
        }
    }
```
Here are the versions that I use:
```
Terraform v0.12.20
+ provider.k8s v0.1.5
+ provider.kubernetes v1.11.1
+ provider.openstack v1.26.0
```

Could you propose a way to hide the values of the data block with Terraform means? 

Thanks!

**Comment by mingfang** [comment]
One options is to move the base64 encoded secret to the `terraform.tfvars` file and then declare it as a variable, e.g. `variable "mysecret"`. Then you can reference the variable in the secret resource like this `"metadata.xml" = var.mysecret`

Does that answer your question?

**Comment by mingfang** [comment]
@UncleIS Thank you raising this important issue.
I fixed the issue in v0.1.6 https://github.com/mingfang/terraform-provider-k8s/releases/tag/v0.1.6
Please give it a try.
Thanks

**Comment by UncleIS** [comment]
Thanks, @mingfang, with version 0.1.6 it works like a charm!

```
Terraform will perform the following actions:                                  
                                                                               
  # k8s_core_v1_secret.app_saml will be created                                
  + resource "k8s_core_v1_secret" "app_saml" {                                 
      + data        = (sensitive value)                                        
      + id          = (known after apply)                                      
      + string_data = (known after apply)                                                                                                                     
      + type        = (known after apply)                                      
                                                                               
      + metadata {                                                             
          + annotations                   = (known after apply)                
          + creation_timestamp            = (known after apply)                
          + deletion_grace_period_seconds = (known after apply)                
          + deletion_timestamp            = (known after apply)                                                                                               
          + labels                        = (known after apply)                
          + name                          = "app-saml"
          + namespace                     = "app"
          + resource_version              = (known after apply)
          + self_link                     = (known after apply)
          + uid                           = (known after apply)
        }
    }
```
